pkg/app/piped/cloudprovider/ecs/ecs.go

```diff
@@ -40,6 +40,7 @@ type Client interface {
 	DeregisterTaskDefinition(ctx context.Context, taskDefinition types.TaskDefinition) (*types.TaskDefinition, error)
 	CreateTaskSet(ctx context.Context, service types.Service, taskDefinition types.TaskDefinition) (*types.TaskSet, error)
 	DeleteTaskSet(ctx context.Context, service types.Service, taskSet types.TaskSet) (*types.TaskSet, error)
+	UpdateServicePrimaryTaskSet(ctx context.Context, service types.Service, taskSet types.TaskSet) (*types.TaskSet, error)
```

Comment:
I just realized that the signature of our functions are full of `passes struct and returns pointers` pattern 😂 Should we make another PR to refactor it as `passes pointers and returns pointers` 👀 Ofcourse will do it after the sync function works perfectly as expected 😄

Reply:
Ok. I understand.